webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js

```diff
@@ -50,14 +50,12 @@ const PageLayout = ({
           <Col className="title_filter" md={searchable ? 6 : 4}>
             {searchable && (
               <div id="search-bar">
-                {
```

Comment:
Does this change have to be here? Doesn't feel related to this PR. Not a biggie though.

Reply:
@ezr-ondrej This was just to fix a merge conflict, not sure why I had to do it.

Reply:
I guess it's some leftover, but I it's not so important, just keep an eye on unrelated changes in the future, no blocker for the merge.